events/wc_druidic_events.txt

```diff
@@ -148,7 +203,96 @@ narrative_event = {
 		}
 	}
 }
-# Suggests liege use worgens against enemies
+# Arugal returns to Gilneas, or not...
```

Comment:
All 4 titles can be ruled by the same character.
In that case, Arugal will have 4 variants to arrive to the court of the same character.